### PR TITLE
aspect-build/bazel-lib@v1.8.0

### PR DESCRIPTION
**Pull request by kormide**
[Automated] Publish aspect-build/bazel-lib@v1.8.0.

### CODE REVIEW

**Comment by meteorcloudy** [comment]
Can you fix the metadata.json conflict?

**Comment by kormide** [comment]
Fixed.

File: modules/aspect_bazel_lib/1.8.0/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"aspect-build/bazel-lib"
```
Comment (fmeum) [inline modules/aspect_bazel_lib/1.8.0/MODULE.bazel:1]: Is this line supposed to be there?

Comment (kormide) [reply modules/aspect_bazel_lib/1.8.0/MODULE.bazel:1]: Yup

**Review by fmeum — APPROVED** [review]

**Pull request merged**